tf/modules/monitoring/metric-alarms.tf

```diff
@@ -1,28 +1,26 @@
 locals {
   alarms = {
-    "es-cluster-health-${var.environment}" = {
-      display_name            = "Elasticsearch Cluster Health Status"
-      filter                  = "metric.type = \"prometheus.googleapis.com/elasticsearch_cluster_health_status/gauge\" AND metric.labels.color = \"green\""
-      comparison              = "COMPARISON_LT"
+    "es-cluster-yellow-${var.environment}" = {
+      display_name            = "Elasticsearch Cluster Health Yellow"
+      filter                  = "metric.type = \"prometheus.googleapis.com/elasticsearch_cluster_health_status/gauge\" AND metric.labels.color = \"yellow\""
+      comparison              = "COMPARISON_GT"
       evaluation_missing_data = "EVALUATION_MISSING_DATA_ACTIVE"
       trigger_count           = 1
-      threshold_value         = 1
-      duration                = "60s"
+      threshold_value         = 0
+      duration                = "900s"
       condition_absent        = "300s"
       min_group_by            = "metric.label.es_cluster"
     },
-    "es-cluster-available-shards-${var.environment}" = {
```

Comment:
I'm happy to see this go; I wonder if we ought to add (in a follow up so not blocking) an alert for the shard size growing beyond a certain size. That would be a comparable alert to this one we are removing for growth in search content we may not have noticed